src/styles/components/advanced-section/styles.less

```diff
@@ -0,0 +1,16 @@
+& when (@advanced-section-enabled) {
+  .advanced-section {
+    &-content {
+      margin-top: @pod-margin-top;
+    }
+
+    &-label,
+    &-label:hover {
+      text-decoration: none;
+
+      .icon {
+        margin-right: @base-spacing-unit * 1/4;
+      }
```

Comment:
Let's adjust this block as follows:

```less
&-label {
  
  &,
  &:hover {
    text-decoration: none;
  }

  .icon {
    margin-right: @base-spacing-unit * 1/4;
  }
}
```
This way we're not unnecessarily duplicating the properties for the nested `.icon` selector.